main.cpp

```diff
@@ -73,31 +73,37 @@ struct <#structName1#>                                //4
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float fval_1 { 0 }, fval_2 { 0 };
+    float ConvergeUVars( float* updatedValue )      //12
     {
-        
+        std::cout << "U's fval_1 value: " << fval_1 << std::endl;
+        fval_1 = *updatedValue;
+        std::cout << "U's fval_1 updated value: " << fval_1 << std::endl;
+        while( std::abs(fval_2 - fval_1) > 0.001f )
+        {
+            fval_2 += 0.5f;  // makes distance between fval_2 and fval_1 get smaller
+        }
+        std::cout << "U's fval_2 updated value: " << fval_2 << std::endl;
+        return fval_2 * fval_1; 
     }
 };
 
-struct <#structname2#>
+struct AddAndMultUvals
 {
-    static <#returntype#> <#staticFunctionA#>(U* that, <#type name#>* <#updatedValue#> )        //10
+    static float ConvergeUvarsStatic( U* that, float* updatedValue )        //10
```

Comment:
Type names begin with a capital letter.
variable names and member function names begin with a lower case letter
- [ ] revise